webapp/package.json

```diff
@@ -64,8 +64,8 @@
     "webpack-cli": "3.3.12"
   },
   "dependencies": {
-    "@primer/octicons-react": "9.6.0",
-    "core-js": "3.6.5",
+    "@primer/octicons-react": "^10.1.0",
+    "core-js": "3.1.4",
```

Comment:
Why does `core-js` need go get downgraded?

Reply:
@hanzei Strange I don't know how it happened. Made this change back.